src/main/java/org/jenkinsci/plugins/ghprb/GhprbRootAction.java

```diff
@@ -170,10 +164,8 @@ private String extractRequestBody(StaplerRequest req) {
         return body;
     }
 
-    
-    private Set<GhprbWebHook> getWebHooks() {
-        final Set<GhprbWebHook> webHooks = new HashSet<GhprbWebHook>();
-
+    private Set<GhprbTrigger> getTriggers(String repoName, String body, String signature) {
+        final Set<GhprbTrigger> triggers = new HashSet<GhprbTrigger>(5);
```

Comment:
I am unsure what would happen to triggers that go away.  I will write a test and see what happens.  I was thinking ConcurrentHashMap<String, WeakHashSet<GhprbTrigger>>